app/views/layouts/darkswarm.html.haml

```diff
@@ -50,4 +33,23 @@
 
           #footer
     %loading
+
+    %script{:src => "https://js.stripe.com/v3/", :type => "text/javascript"}
+    %script{src: "//maps.googleapis.com/maps/api/js?libraries=places,geometry#{ ENV['GOOGLE_MAPS_API_KEY'] ? '&key=' + ENV['GOOGLE_MAPS_API_KEY'] : ''} "}
+    = javascript_include_tag "darkswarm/all"
+    = javascript_include_tag "web/all"
+    = render "layouts/i18n_script"
+    = yield :scripts
+
+    = inject_current_hub
+    = inject_json "user", "current_user"
+    = inject_json "railsFlash", "flash"
+    = inject_taxons
+    = inject_properties
+    = inject_current_order
+    = inject_currency_config
+    = yield :injection_data
+
+    = render "layouts/bugherd_script"
+    = render "layouts/matomo_tag"
     = render 'spree/shared/google_analytics'
```

Comment:
- from what I read google analytics will not report partial page loads if the tag is not in the head. maybe the same for matomo. I wouldnt worry about this, at our scale better have a fast page and a little incorrect stats